src/helpers/section.ts

```diff
@@ -0,0 +1,62 @@
+import Cell from '../Interfaces/model/cell.interface';
+import SectionModel from '../Models/section.model';
+
+/**
+ * create a new section with no cells
+ * @param cells cells to add to section
+ */
+export const createSection = async (cells?: Cell[]) => {
+    return await SectionModel.create({ cells: cells ? cells : [] });
```

Comment:
A section should have atleast one cell. So create a cell and push it to section?